.github/workflows/cypress-tests-runner.yml

```diff
@@ -128,7 +128,7 @@ jobs:
           CONNECTOR_AUTH_PASSPHRASE: ${{ secrets.CONNECTOR_AUTH_PASSPHRASE }}
           CONNECTOR_CREDS_S3_BUCKET_URI: ${{ secrets.CONNECTOR_CREDS_S3_BUCKET_URI}}
           DESTINATION_FILE_NAME: "creds.json.gpg"
-          S3_SOURCE_FILE_NAME: "aa328308-b34e-41b7-a590-4fe45cfe7991.json.gpg"
+          S3_SOURCE_FILE_NAME: "ae17cddf-4e13-471e-a78d-58eafcfd66a5.json.gpg"
```

Comment:
can you please fetch the recent one (take 71, see description) and update the creds? this one has a lot of discrepancies and would result in all tests to fail.